BUILD

```diff
@@ -1858,6 +1858,7 @@ grpc_cc_library(
         "grpc_resolver_fake",
         "grpc_resolver_dns_native",
         "grpc_resolver_sockaddr",
+        "grpc_resolver_binder",
```

Comment:
Does this mean that we are linking Java code into all gRPC binaries?  Is that really what we want?  Maybe this should be a separate BUILD target that can be added only when the application needs it?